update/state/state.go

```diff
@@ -205,6 +210,7 @@ func estimateSequence(url string, timestamp time.Time) int {
 		}
 	}
 
-	behind := state.Time.Sub(timestamp)
-	return state.Sequence - int(behind.Minutes())
+    behind := state.Time.Sub(timestamp)
+    // Sequence unit depends on replication interval (minute, hour, day).
+    return state.Sequence - int(math.Ceil(behind.Minutes() / config.BaseOptions.ReplicationInterval.Minutes()))
```

Comment:
To be consistent with the current convention of `url` being passed into the function instead of being read directly from the config, `interval` should also be passed in to this function rather than accessed directly from the config. Reading directly from config for some params but not others is confusing.